modules/waf/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "acl" {
```

Comment:
prob update the name of the output since it outputs the entire an AWS Web Application Firewall, not just ACL